collect_app/src/main/java/org/odk/collect/android/formentry/FormEntryViewModel.java

```diff
@@ -118,12 +118,9 @@ public void promptForNewRepeat() {
             return;
         }
 
-        worker.immediate((Supplier<Void>) () -> {
-            jumpBackIndex = formController.getFormIndex();
-            jumpToNewRepeat();
-            updateIndex();
-            return null;
-        }, ignored -> {});
+        jumpBackIndex = formController.getFormIndex();
+        jumpToNewRepeat();
+        refreshSync();
```

Comment:
Can we switch all these internal `refreshSync` calls to `updateIndex(false)`?

Reply:
Yes.